frontend/src/metabase/components/Subhead.jsx

```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+const Subhead = styled.h2``;
+
+export default Subhead;
```

Comment:
One thing I’d like to do is add more of these components to the style guide with descriptions of where to use them. Currently I have to just pick a h* closest to the designs which leads to inconsistencies.